test/e2e/set/scyllacluster/scyllacluster_scaling.go

```diff
@@ -55,7 +63,15 @@ var _ = g.Describe("ScyllaCluster", func() {
 		sc, err = waitForScyllaClusterState(waitCtx2, f.ScyllaClient().ScyllaV1(), sc.Namespace, sc.Name, scyllaClusterRolledOut)
 		o.Expect(err).NotTo(o.HaveOccurred())
 
-		verifyScyllaCluster(ctx, f.KubeClient(), sc)
+		verifyScyllaCluster(ctx, f.KubeClient(), sc, d)
+
+		framework.By("Inserting data with RF=1")
+		d, err = NewDataInserter(ctx, f.KubeClient().CoreV1(), sc, 1)
+		defer d.Close()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		err = d.Insert()
```

Comment:
should we verify the new data before scaling? (so we always now which part went wrong)